docs/application/dotnet/samples/tizen-samples.md

```diff
@@ -31,6 +31,18 @@
 			<p>SecureRepository is simple application that allows you to securely store data, certificates and cryptographic keys using C# Tizen.Security.SecureRepository API.</p>
```

Comment:
Recommend some minor edits:
SecureRepository is a simple application that allows you to securely store data, certificates, and cryptographic keys using the Tizen.Security.SecureRepository C# API.